src/huggingface_hub/templates/modelcard_template.md

```diff
@@ -81,15 +81,20 @@ Use the code below to get started with the model.
 
 {{ training_data | default("[More Information Needed]", true)}}
 
-## Training Procedure [optional]
+## Training Procedure 
 
 <!-- This relates heavily to the Technical Specifications. Content here should link to that section when it is relevant to the training procedure. -->
 
-### Preprocessing
+### Preprocessing [optional]
 
 {{ preprocessing | default("[More Information Needed]", true)}}
 
-### Speeds, Sizes, Times
+
+### Training hyperparameters
```

Comment:
Would like new line here + capital letters on title to keep with pattern

```suggestion
### Training Hyperparameters

```

Reply:
Also, if this section is called "Training hyperparameters" shouldn't other info be shown/suggested here? Just a thought.